src/schema/users.ts

```diff
@@ -375,8 +375,7 @@ export const resolvers: IResolvers<any, Context> = {
           : null,
       };
     },
-    userInfo: async (_, { id }: ReadingRankArgs): Promise<GQLUser> =>
-      fetchUserById(id),
+    user: (_, { id }: ReadingRankArgs): Promise<GQLUser> => fetchUserById(id),
```

Comment:
I think this is better

```suggestion
    user: (_, { id }:{id: string): Promise<GQLUser> => fetchUserById(id),
```